maquetado/setup/scss/tech.scss

```diff
@@ -0,0 +1,49 @@
+$tech-logo-size: 100px;
+$tech-container-margin: 50px;
+$tech-container-height: 300px;
+$tech-container-width: 210px;
+
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: $tech-container-height;
+  margin: $tech-container-margin;
```

Comment:
Margins are usually more customizable, I do not recommend to use constants for this